test/test_quantization.py

```diff
@@ -36,6 +36,7 @@
 from quantization.core.test_workflow_module import TestHistogramObserver  # noqa: F401
 from quantization.core.test_workflow_module import TestDistributed  # noqa: F401
 from quantization.core.test_workflow_module import TestFusedObsFakeQuantModule  # noqa: F401
+from quantization.core.test_config import TestBackendConfig  # noqa: F401
```

Comment:
nit: test_config -> test_backend_config here as well

Reply:
Oops I'll fix that